frontend/packages/kubevirt-plugin/src/utils/index.ts

```diff
@@ -8,7 +9,34 @@ export const getBasicID = <A extends K8sResourceKind = K8sResourceKind>(entity:
 export const prefixedID = (idPrefix: string, id: string): string =>
   idPrefix && id ? `${idPrefix}-${id}` : null;
 
+export const joinIDs = (...ids: string[]): string => ids.join('-');
```

Comment:
FYI, the `string` return type is unnecessary, since TS can infer it from the function's body.

Reply:
I just thought it would make more clear interface if anyone was about to change the function. I guess it can be removed, because they are just simple methods.

Reply:
I'm OK with explicit return type too.

If someone changes the function signature (either explicitly or implicitly by changing the impl. with TS auto-inferring return type), other code where it's used should report type errors, since the return type expectation was changed.

Reply:
ah, that is true